app/packages.json

```diff
@@ -1,8 +1,9 @@
 {
     "packages": [],
     "includes": {
-        "include/all$4e9a2d9d7ec77a574c121dcc927caf506c40c258.json": {
-            "sha1": "4e9a2d9d7ec77a574c121dcc927caf506c40c258"
+        "include/all$920cacf6dd96849dd08db78d9cb7c9deb51bf8c9.json": {
+            "sha1": "920cacf6dd96849dd08db78d9cb7c9deb51bf8c9"
         }
-    }
+    },
+    "metadata-url": "/p2/%package%.json"
```

Comment:
Previous Satis update PRs have excluded `metadata-url` and the accompanying `.json` files.. is this intentional? It's the reason why this PR is so.. hefty.

Reply:
To date the PRs simply contain the output generated by the [satis app](https://github.com/composer/satis). I've just googled `metadata-url` but I'm not wiser about the purpose of it.

Reply:
Unfortunately I don't have any context of this change, perhaps @simesy or @tobybellwood might be able to answer why this was excluded previously.

Reply:
Yeah, this is just the output from `ahoy build stable`, looks like this is intentional for composer 2.0 support: https://github.com/composer/satis/blob/master/src/Builder/PackagesBuilder.php